### PR TITLE
Adds new amp-story zoom/pan animations ✨ 

### PR DESCRIPTION
**Pull request by Enriqe**
Solves #12173

Adds new amp-story zoom/pan animations that allows users to achieve a "Ken Burns" effect with images. Includes examples testing usage.

We might want to add an amp-by-example description so that users trying to achieve the effect can find the instructions on how to do it somewhere.

### CODE REVIEW

File: extensions/amp-story/0.1/animation-presets.js [hunk 1]
```diff
@@ -179,4 +179,56 @@ export const PRESETS = {
...
+    duration: 30000,
+    easing: 'linear',
+    keyframes(dimensions) {
+      const offsetX = -dimensions.targetWidth / 2;
+      const offsetY = dimensions.pageHeight - dimensions.targetHeight;
+      return translate2d(offsetX, offsetY, offsetX, 0);
+    }
+  },
+  'zoom-in': {
+    duration: 30000,
```
Comment (alanorozco) [inline extensions/amp-story/0.1/animation-presets.js]: nit: `30s` makes sense for a Ken Burns effect, but for an individual preset I think `zoom-in` should be set to something between 500-1000 `ms`.

File: extensions/amp-story/0.1/animation-presets.js [hunk 2]
```diff
@@ -179,4 +179,56 @@ export const PRESETS = {
...
+      const offsetY = dimensions.pageHeight - dimensions.targetHeight;
+      return translate2d(offsetX, offsetY, offsetX, 0);
+    }
+  },
+  'zoom-in': {
+    duration: 1000,
+    easing: 'linear',
+    keyframes: [
+      {transform: 'scale(1,1)'},
+      {transform: 'scale(3,3)'}
```
Comment (gmajoulet) [inline extensions/amp-story/0.1/animation-presets.js]: As seen offline, we might want to experiment a bit with that and see if we're happy with the image quality :)

Comment (Enriqe) [reply extensions/amp-story/0.1/animation-presets.js]: Yep, will speak with Hong from UX today to see what she thinks 👍

**Comment by Enriqe** [comment]
After discussion with UX we decided to keep the default values for the animation as is, to allow more flexibility.

We will eventually write doc / example on how to achieve this effect with optimal results. The documentation will describe the practices to achieve the best results, including:

- Image sizing

- Animation duration

- Order of animation (panning in img and zooming in img-container vs. doing it a different way).

**Pull request merged**